components/metris/README.md

```diff
@@ -18,12 +18,16 @@ Metris is a metering component that collects data and sends them to EDP.
 | `--edp-buffer` | **EDP_BUFFER** | Number of events that the buffer can have | `100` |
 | `--edp-workers` | **EDP_WORKERS** | Number of workers to send metrics | `5` |
 | `--edp-event-retry` | **EDP_RETRY** | Number of retries for sending an event | `5` |
-| `--provider-poll-interval` | **PROVIDER_POLLINTERVAL** | Interval at which metrics are fetched | `1m` |
+| `--provider-poll-interval` | **PROVIDER_POLLINTERVAL** | Interval at which metrics are fetched | `5m` |
+| `--provider-poll-max-interval` | **PROVIDER_POLLMAXINTERVAL** | maximum Interval at which metrics are fetch | `15m` |
 | `--provider-poll-duration` | **PROVIDER_POLLDURATION** | Time limit for requests made by the provider client | `5m` |
+| `--provider-max-retries` | **PROVIDER_MAXRETRIES** | Number of maximum retries until a cluster is removed from the cache in case of not being found on the provider. This will stop sending events for the removed cluster! | `20` |
```

Comment:
```suggestion
| `--provider-max-retries` | **PROVIDER_MAXRETRIES** | Maximum number of retries before a cluster is removed from the cache if it is not found on the provider. NOTE: This will stop sending events for the removed cluster | `20` |
```

It's not a big deal, but we normally avoid exclamation points and use "Note" or "Caution" [instead](https://kyma-project.io/community/guidelines/content/#formatting-formatting-panels).